packages/react-core/src/components/Card/CardHeader.tsx

```diff
@@ -75,6 +75,8 @@ export interface CardHeaderProps extends React.HTMLProps<HTMLDivElement> {
   toggleButtonProps?: any;
   /** Whether to right-align expandable toggle button */
   isToggleRightAligned?: boolean;
+  /** Flag indicating that header wrapping is enabled */
+  hasHeaderWrap?: boolean;
```

Comment:
More of a nit, but we could just call this `hasWrap` instead. Don't think I'd block over it.